Holovibes/mainwindow.ui

```diff
@@ -1045,6 +1072,9 @@
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="PLabel">
+             <property name="toolTip">
+              <string>The Time Transform filter's start frequency</string>
```

Comment:
The Time Transform FFT filter's start frequency : the range of frequencies kept is [z, z + width].